package.json

```diff
@@ -26,7 +26,7 @@
     "app-devserver": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly {1}\" yarn:app-python-devserver yarn:hashi-dev --",
     "devserver": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly {1}\" yarn:python-devserver yarn:hashi-dev --",
     "devserver-hot": "concurrently --passthrough-arguments --kill-others \"yarn:watch-hot --watchonly {1}\" yarn:python-devserver yarn:hashi-dev --",
-    "devserver-with-kds": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly --kds-path={1}\" yarn:python-devserver yarn:hashi-dev --",
+    "devserver-with-kds": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly={2} --require-kds-path --kds-path={1}\" yarn:python-devserver yarn:hashi-dev --",
```

Comment:
Non-blocking as I think this is probably out of scope of the issue you're trying to resolve here, so I'll make a follow-up issue after merging this.

But - I think that we'll need to do some tweaking because when I use the `--watchonly` flag here with:

`yarn devserver-with-kds path/to/kds user_auth` then the build system doesn't recognize whenever I make a change in KDS.

But when I don't pass in any plugins for the `--watchonly` then the front-end rebuilds reactively to changes I make within KDS.

Reply:
I understand that. And yes, it'd be better to have a follow-up for this issue so that it can be focussed on separately. I too am not sure how to move forward with it. I believe this is one important feature to have and requires separate attention.